android/tangram/src/com/mapzen/tangram/MapView.java

```diff
@@ -65,7 +81,11 @@ public void getMapAsync(@NonNull final OnMapReadyCallback callback,
             @SuppressWarnings("WrongThread")
             protected Boolean doInBackground(Void... params) {
                 mapInstance.init();
-                mapInstance.loadSceneFile(sceneFilePath);
+                if (sceneUpdates.size() > 0) {
```

Comment:
This branch should be unnecessary if we allow empty or `null` lists as the second argument to `loadSceneFile`.